scipp/qens-reduction.ipynb

```diff
@@ -0,0 +1,409 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a146780d-97b5-45a2-afe3-b852551bee2f",
+   "metadata": {},
+   "source": [
+    "# QENS data reduction - DMSC summer school"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9d2fb1a1-e0ea-4730-9a5a-0e5f8d6c6cd2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import plopp as pp\n",
+    "import scipp as sc\n",
+    "import scippneutron as scn\n",
+    "import utils"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "45e298f2-8c63-492d-a4fc-84fd6eee81c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib widget"
```

Comment:
This messes up the plots in the book (only the toolbar shows up. We should have static figs in the book I think.

Reply:
Ok. I needed the zoom functionality to find a good mask range though.

Reply:
Sure, we can always tell them how to make plots interactive during the session.